doc/dev_guide/building_from_source.rst

```diff
@@ -38,43 +38,43 @@ the preferences can differ. But strategically the steps are always the same.
      option in step 5. For all platforms, this is ``python`` version 2.7+
      (but not 3.x). You need the following Python modules:
 
-       * `pyyaml <https://pypi.python.org/pypi/PyYAML>`_ version 3.10
-       * `argparse <https://pypi.python.org/pypi/argparse>`_ version 1.1
-       * `msgpack-python <https://pypi.python.org/pypi/msgpack-python>`_ version 0.4.6
-       * `gevent <https://pypi.python.org/pypi/gevent>`_ version 1.1.2
-       * `six <https://pypi.python.org/pypi/six>`_ version 1.8.0
+     * `pyyaml <https://pypi.python.org/pypi/PyYAML>`_ version 3.10
+     * `argparse <https://pypi.python.org/pypi/argparse>`_ version 1.1
+     * `msgpack-python <https://pypi.python.org/pypi/msgpack-python>`_ version 0.4.6
+     * `gevent <https://pypi.python.org/pypi/gevent>`_ version 1.1.2
+     * `six <https://pypi.python.org/pypi/six>`_ version 1.8.0
```

Comment:
* argparse — it is built-in in Python 2.6+ AFAIR.
* msgpack-python is a submdoule of test-run, so it is not necessary to install it separately.
* All other versions seems obsoleted. And there is not any sense to list them, they're listed in `test-run/requirements.txt` in the format that pip understands. One who want to install it using a package manager can find related package names in `rpm/tarantol.spec` or `debian/control` (but I would left this exercise to a user, just mention where to find it).